doc/nrf/shortcuts.txt

```diff
@@ -202,3 +202,6 @@
 .. |no_changes_yet_note| replace:: No changes since the latest |NCS| release.
 
 .. |serial_port_sec_tag| replace:: where ``<serial port>`` is the serial port of your device and ``<sec tag>`` is the previously chosen unused security tag.
+
+.. |hex_file| replace:: The certificate is automatically converted to a HEX file in :file:`CMakeLists.txt`.
```

Comment:
It could be a bit ambiguous to call it a hex file, as it differs from the hex file format we usually work with. The output file is a `.inc` file that has the content of the certificate file converted to a list of hexadecimal characters

Reply:
Updated the text as per above comment. Please check.

Reply:
Updated.